products/jenkins.md

```diff
@@ -27,12 +27,20 @@ releases:
     latestReleaseDate: 2023-03-14
     releaseDate: 2016-04-20
 
+-   releaseCycle: "2.387"
+    lts: true
+    support: 2023-03-08
+    eol: false
+    latest: "2.387.1"
+    latestReleaseDate: 2023-03-08
+    releaseDate: 2023-03-08
+
 -   releaseCycle: "2.375"
     lts: true
     support: 2022-10-25
     eol: false
     latest: "2.375.4"
-    latestReleaseDate: 2023-03-06
+    latestReleaseDate: 2023-03-08
```

Comment:
This change will be reverted by our automation.

Our automation is using the tag date, not the GitHub release date. It's wrong but, for the time being, this is not possible to manually "force" the date (unless we disable the automation).

Will have to check (not in this PR) if the GitHub releases automation script I wrote for adoptium/temurin could be used here. It could fix the issue.

Reply:
No problem.  The tag for security releases of Jenkins core are created on a private repository a few days before the actual release.  That allows the release to be staged privately without risking premature disclosure of the vulnerabilities that are being fixed.